packages/omf/tests/exporter/test_evo_attribute_to_omf.py

```diff
@@ -45,16 +44,16 @@
     Vector3Data,
 )
 
-from evo.data_converters.common import EvoWorkspaceMetadata, create_evo_object_service_and_data_client
+from evo.data_converters.common import create_evo_object_service_and_data_client
 from evo.data_converters.omf.exporter import export_attribute_to_omf
 from evo.data_converters.omf.importer import convert_omf
 from evo.data_converters.omf.importer.omf_attributes_to_evo import int_to_rgba
+from evo.data_converters.common.test_support import EvoStubMixin
 
 
-class TestOMFAttributeExporter(TestCase):
+class TestOMFAttributeExporter(EvoStubMixin, TestCase):
```

Comment:
`EvoStubMixin` indirectly inherits from `TestCase` as both `TestWithConnector`, and `TestWithStorage` are subclasses of `TestCase`. So specifying `TestCase` isn't needed.

Probably a better name for `EvoStubMixin` is EvoDataConvertersTestCase` or something similar.